src/Apps/W1/PEPPOL/App/src/Interfaces/PEPPOL30ExportManagement.Interface.al

```diff
@@ -0,0 +1,63 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.Peppol;
+
+using Microsoft.Finance.VAT.Calculation;
+using Microsoft.Finance.VAT.Setup;
+using Microsoft.Foundation.Attachment;
+using Microsoft.Sales.Document;
+
+/// <summary>
+/// Interface for managing PEPPOL 3.0 export operations.
+/// Provides methods for initializing export data, iterating through records and lines,
+/// and calculating totals for PEPPOL document generation.
+/// </summary>
+interface "PEPPOL30 Export Management"
```

Comment:
This is a very weird interface.
1 It has management in the name, which means it has no clear purpose. 
2 it is iterating over the records.
3 I would expect it has something to do with export

Maybe initialize and export, I wonder why is it a need for get operations and etc.
should it be internal?